.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     name: Run pipeline with test data
     # Only run on push if this is the nf-core dev branch (merged PRs)
     if: "${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'sanger-tol/treeval') }}"
-    runs-on: ubuntu-latest
+    runs-on: ubuntu2204-16c
```

Comment:
Can you try `ubuntu2204-4c` first with 4 core and 16G memory first, if not try 8c, 16c and 32c.

Reply:
4c failed but not because the memory:
`2023-12-05T17:01:10.4799617Z Access to 'PRETEXTMAP_HIGHRES.out' is undefined since the process 'PRETEXTMAP_HIGHRES' has not been invoked before accessing the output attribute`

Reply:
Yes, i commented out the wrong line. Still failing due to broken pipe.